public/js/main.js

```diff
@@ -70,4 +70,36 @@ $(document).ready(function(){
             result.add("<img class='img-place  mr-10' src='" + imgpath + "'>").appendTo('#showImage');
         });
     });
+
+    /**
+     * remove image when click button remove image
+     */
+    $('.btn-remove-img').bind('click',function(e){
+        e.preventDefault();
+        var img_comtainer = $('#old-images');
+        var remove_id = $(this).parent().attr('id');
```

Comment:
cái này nên là `var img_item = $(this).parent();` rồi ở dưới thì remove ri thôi: `img_item.remove()`